src/Illuminate/Support/Collection.php

```diff
@@ -138,6 +138,29 @@ public function every($step, $offset = 0)
     }
 
     /**
+     * Fills a collection with items.
+     *
+     * @param  int  $amount
+     * @param  mixed  $value
+     * @param  bool  $append
+     * @return static
```

Comment:
static implies it's a new instance of the class, but $this means it's the same instance